processor.py

```diff
@@ -126,7 +126,7 @@ def inverse(self):
                 cofact_r = []
                 for c in range(len(M)):
                     minor = self.get_minor(M, r, c)
-                    cofact_r.append((-1 if (r + c) % 2 else 1) * self.det_recur(minor))
+                    cofact_r.append(-1 ** ((r + c) % 2) * self.det_recur(minor))
```

Comment:
Originally, we used your version for this, but exponentiation is much slower than a ternary operator, so we switched to this. I'd suggest reverting this change.